src/dialogs/config-entry-system-options/dialog-config-entry-system-options.ts

```diff
@@ -31,18 +29,21 @@ class DialogConfigEntrySystemOptions extends LitElement {
 
   @internalProperty() private _disableNewEntities!: boolean;
 
+  @internalProperty() private _open = false;
+
   @internalProperty() private _error?: string;
 
   @internalProperty() private _params?: ConfigEntrySystemOptionsDialogParams;
 
   @internalProperty() private _loading?: boolean;
 
-  @internalProperty() private _submitting?: boolean;
+  @internalProperty() private _submitting = false;
 
   public async showDialog(
```

Comment:
```suggestion
  public showDialog(
```

Reply:
Shouldn't this be an async function because the call to `getConfigEntrySystemOptions` in the `showDialog` method returns a promise?

Reply:
Oh, yeah it should